src/parser/FStar.Parser.Dep.fs

```diff
@@ -709,6 +712,17 @@ let collect_one
         collect_term t
     | Attributes cattributes  ->
         List.iter collect_term cattributes
+    | CalcProof (rel, init, steps) ->
+        (* TODO: add the Calc module too *)
```

Comment:
What's this remark about? In the next line, you correctly add FStar.Calc

Reply:
I forgot to remove the comment :)